### PR TITLE
Add soaring options

### PR DESCRIPTION
**Pull request by byanofsky**
Adding options to soar
Please review this, @casey-collab

### CODE REVIEW

**Comment by byanofsky** [comment]
Please review this, @casey-collab

File: english/soar/soar.md [hunk 1]
```diff
@@ -3,4 +3,7 @@ No, not like a bird, like a jet!
 
 With the ground moving rapidly past you.
 
-Do you:
\ No newline at end of file
+Do you:
+
+[Fly to Paris] (fly-to-paris/fly-to-paris.md)
+[Fly to London] (fly-to-london/fly-to-london.md)
```
Comment (casey-collab) [inline english/soar/soar.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by byanofsky** [comment]
Changes made @casey-collab

File: english/soar/soar.md [hunk 2]
```diff
@@ -3,4 +3,10 @@ No, not like a bird, like a jet!
...
 
-Do you:
\ No newline at end of file
+Do you:
+
+[Fly to Paris] (fly-to-paris/fly-to-paris.md)
+
+[Fly to London] (fly-to-london/fly-to-london.md)
+
+[Change that wasn't pushed] (change-not-pushed)
```
Comment (casey-collab) [inline english/soar/soar.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by byanofsky** [comment]
Changes made @casey-collab

**Comment by casey-collab** [comment]
Looks great! :shell:

**Pull request merged**